packages/fx-core/src/component/driver/teamsApp/createAppPackage.ts

```diff
@@ -205,6 +225,25 @@ export class CreateAppPackageDriver implements StepDriver {
     return ok(new Map());
   }
 
+  private static async expandOpenAPIEnvVars(
+    openAPISpecPath: string,
+    ctx: WrapDriverContext
+  ): Promise<Result<string, FxError>> {
+    const content = await fs.readFile(openAPISpecPath, "utf8");
+    const vars = getEnvironmentVariables(content);
+    ctx.addTelemetryProperties({
+      [TelemetryPropertyKey.customizedOpenAPIKeys]: JSON.stringify(vars),
```

Comment:
Better to use vars.join(";")

The suggested value for array is like "value1;value2;value3"